modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -763,6 +765,63 @@ public abstract class Base${schemaName}ResourceImpl
 				throw new UnsupportedOperationException("This method needs to be implemented");
 			</#if>
 		}
+
+		<#if (privateParsersNeeded?size >= 1)>
+			<#if privateParsersNeeded?seq_contains("boolean")>
+				private Boolean _parseBoolean(String value){
+					Boolean booleanValue = null;
+					if (value != null){
+						booleanValue = Long.parseBoolean(value);
```

Comment:
`Long.parseBoolean` :thinking:

Reply:
I had already resolved this one. Editing without a ftl highlighter is dificult.